Makefile

```diff
@@ -14,9 +19,9 @@ NODE_BIN := $(THIS_DIR)/node_modules/.bin
 
 # applications
 NODE ?= node
-NPM ?= $(NODE) $(shell which npm)
+NPM ?= npm
```

Comment:
Ah, I see. But calling `node C:\path\to\npm` fails on Windows - because `npm` is a bash script there (also available in `cmd` version, which is just as useful/less). While on Linux/OS X it's a nodejs script - go figure. But there's no guarantee that won't change, so it's best to just call `npm` directly and let `nvm`/whatever (or just the user who wants `exotic-node`) figure out the right `node`/`npm` combo. If needed, `NODE` and `NPM` variables can be provided to `make`.